lox_services/persistence/database/insert.py

```diff
@@ -150,6 +152,10 @@ def insert_dataframe_into_database(
         dataset = "Utils"
         if table.name == "CurrencyConversion":
             dataframe = remove_duplicate_currency_conversion(dataframe)
+    elif isinstance(table, CarrierData_dataset):
+        dataset = "CarrierData"
+        if table.name == "PackageInformation":
+            dataframe = remove_duplicate_package_information(dataframe)
```

Comment:
it would be nice to also add a quality_check function here 
checking at least null value + valid country code